.github/workflows/cekit-action.yaml

```diff
@@ -0,0 +1,51 @@
+name: Publish cekit
+on:
+  workflow_dispatch
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Install CEKit
+        uses: cekit/actions-setup-cekit@v1.1.7
+
+      - name: Create folders
+        run: |
+          mkdir -p container-images/jdk17/runtime/
```

Comment:
We could use a matrix to run the same steps for either `jdk17` or `jdk21`, that'd simplify this file.